source/scenario/mainMenu.vwf.yaml

```diff
@@ -17,7 +17,7 @@ extends: ../scenario/scenario.vwf
 properties:
   scenarioName: mainMenuScenario
 
-  nextScenarioPath: "introScreenScenario"
+  nextScenarioPath: "scenario1a"
```

Comment:
Ummmm... is this right?  More to the point, is this scenario even being used any more?  Maybe we should get rid of it completely?